AWSAppSyncClient/Internal/AWSAppSyncRetryHandler.swift

```diff
@@ -76,7 +80,27 @@ final class AWSAppSyncRetryHandler {
     /// Returns a delay in milliseconds for the current attempt number. The delay includes random jitter as
     /// described in https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/
     static func retryDelayInMillseconds(for attemptNumber: Int) -> Int {
-        let delay = Int(Double(truncating: pow(2.0, attemptNumber) as NSNumber) * 100.0 + Double(AWSAppSyncRetryHandler.getRandomBetween0And1() * AWSAppSyncRetryHandler.JITTER))
+        let delay = Int(Double(truncating: pow(2.0, attemptNumber) as NSNumber) * 100.0 + Double(AWSAppSyncRetryHandler.getRandomBetween0And1() * AWSAppSyncRetryHandler.jitterMilliseconds))
         return delay
     }
+
+    /// Returns the value of the "Retry-After" header as an Int, or nil if the value isn't present or cannot be converted to
+    /// an Int
+    ///
+    /// - Parameter response: The response to get the header from
+    /// - Returns: The value of the "Retry-After" header, or nil if not present or not convertable to Int
+    private static func getRetryAfterHeaderValue(from response: HTTPURLResponse) -> Int? {
+        let waitTime: Int?
+        switch response.allHeaderFields["Retry-After"] {
```

Comment:
Not particular related to this change, but I think we should consider doing lowercase/ uppercase for this header to make sure we don't miss this information due to case.

Reply:
No need, the `allHeaderFields` dictionary keys are [case-insensitive](https://developer.apple.com/documentation/foundation/httpurlresponse/1417930-allheaderfields)